### DISCUSSION TITLE
Welcome to HOMER Discussions!

### DISCUSSION THREAD

**Discussion by lmangani** [sipcapture/homer#431] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jmillican61** [comment]


**Reply by jmillican61** [reply]
Well obviously I did not post my logs correctly, apologies for the strike through.

**Reply by adubovikov** [reply]
I am not sure this is related to "Discussions", if you have an issue, please open it in the area that it exists for....https://github.com/sipcapture/homer/issues
